<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>p_awindow</class>
 <widget class="QMainWindow" name="p_awindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1100</width>
    <height>780</height>
   </rect>
  </property>
  <property name="mouseTracking">
   <bool>true</bool>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="mouseTracking">
    <bool>true</bool>
   </property>
   <widget class="QPushButton" name="button_retract">
    <property name="geometry">
     <rect>
      <x>890</x>
      <y>60</y>
      <width>151</width>
      <height>71</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 75 11pt &quot;等线&quot;;</string>
    </property>
    <property name="text">
     <string>悔棋</string>
    </property>
   </widget>
   <widget class="QPushButton" name="button_return">
    <property name="geometry">
     <rect>
      <x>890</x>
      <y>400</y>
      <width>151</width>
      <height>101</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 75 11pt &quot;等线&quot;;</string>
    </property>
    <property name="text">
     <string>返回主菜单</string>
    </property>
   </widget>
   <widget class="QPushButton" name="button_restart">
    <property name="geometry">
     <rect>
      <x>890</x>
      <y>250</y>
      <width>151</width>
      <height>101</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 75 11pt &quot;等线&quot;;</string>
    </property>
    <property name="text">
     <string>重新开始</string>
    </property>
   </widget>
   <widget class="QPushButton" name="button_start">
    <property name="geometry">
     <rect>
      <x>910</x>
      <y>580</y>
      <width>101</width>
      <height>71</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 75 11pt &quot;等线&quot;;</string>
    </property>
    <property name="text">
     <string>开始</string>
    </property>
   </widget>
   <widget class="QPushButton" name="button_banhint">
    <property name="geometry">
     <rect>
      <x>890</x>
      <y>170</y>
      <width>151</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 75 11pt &quot;等线&quot;;</string>
    </property>
    <property name="text">
     <string>开启禁手提示</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1100</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
